pulpcore/cli/python/remote.py

```diff
@@ -98,6 +98,12 @@ def remote(ctx: click.Context, pulp_ctx: PulpCLIContext, /, remote_type: str) ->
         callback=load_json_callback,
         needs_plugins=[PluginRequirement("python", specifier=">=3.2.0")],
     ),
+    pulp_option(
+        "--provenance",
+        type=click.BOOL,
```

Comment:
Shouldn't this be "--provenance/--no-provenance" with default=None,
so it uses server default values and does not change the value without being specified?